src/instruments/src/SD/StatusArea/StatusArea.jsx

```diff
@@ -0,0 +1,108 @@
+/*
+ * A32NX
+ * Copyright (C) 2020 FlyByWire Simulations and its contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import './StatusArea.scss';
+import { Text } from '../Text/Text.jsx';
+import { useGlobalVar, getSimVar } from '../../util.mjs';
+
+export const StatusArea = () => {
+    const gw = getSimVar('TOTAL WEIGHT', 'kg');
+    const zulu = useGlobalVar('ZULU TIME', 'seconds');
+    let tat = Math.round(getSimVar('TOTAL AIR TEMPERATURE', 'celsius'));
+    if (tat > 99 || tat < -99) {
+        tat = tat > 99 ? 99 : -99;
+    }
+    let sat = Math.round(getSimVar('AMBIENT TEMPERATURE', 'celsius'));
+    if (sat > 99 || sat < -99) {
+        sat = sat > 99 ? 99 : -99;
+    }
+    const adirsState = getSimVar('L:A320_Neo_ADIRS_STATE', 'Enum');
+    const satPrefix = sat > 0 ? '+' : '';
+    const tatPrefix = tat > 0 ? '+' : '';
```

Comment:
Since you're only use these once, I'd move them down inside the component next to `{sat}` and `{tat}`.